### ISSUE TITLE
Command to Limit the Queue count & Max Video lenght

### ISSUE THREAD

**Issue by Nanabell** [Frederikam/FredBoat#39]
The idea seems quite good. 
i duno if you want to wait for the dashboard to be done with this since its on the lower end of the priority  list

**Comment by Frederikam** [comment]
Yeah I'm working on the dashboard right now

**Comment by Baschdel-98** [comment]
I'm sorry for beeing anoying but isn't that implmented also?

**Comment by Frederikam** [comment]
yes, however the branch never reached production as we decided not to use MongoDB after all.
